p4include/dpdk/psa.p4

```diff
@@ -57,10 +47,16 @@ limitations under the License.
  *
  * Note that the width of typedef <name>Uint_t will always be the same
  * as the width of type <name>_t. */
-typedef bit<32> PortIdUint_t;
-typedef bit<32> MulticastGroupUint_t;
-typedef bit<16> CloneSessionIdUint_t;
-typedef bit<8>  ClassOfServiceUint_t;
+
+/* WARNING!  Andy Fingerhut made up the following numbers for the DPDK
+ * target data plane bit widths.  No one should believe they are
+ * accurate until they * have been approved by the p4c-dpdk
+ * implementers. */
+
+typedef bit<32>  PortIdUint_t;
```

Comment:
@hanw Do you want DPDK data plane type for PortId_t to be 32 bits wide?  OK if so, just making sure that is what you intended.

The const values for the CPU and recirculate ports in the original psa.p4 include file are simply the numeric values that are used by BMv2, and are also intended to be customized for each PSA implementation, and should be as many bits wide as this type is.

Reply:
I sent an email to Cristian to confirm what would be a good value for PortId_t and whether non-power-of-2 width are okay for DPDK.